gui/executionGraphGui/client/apps/eg/src/app/services/GraphManipulationServiceBinaryHttp.ts

```diff
@@ -17,8 +17,8 @@ import { ILogger, LoggerFactory, stringify } from '@eg/logger';
 import { GraphManipulationService, sz } from './GraphManipulationService';
 import { BinaryHttpRouterService } from './BinaryHttpRouterService';
 import { Node, NodeId, Socket, Connection, fromConnection } from '../model';
-import * as conversions from './Conversions';
-import { isDefined } from '@angular/compiler/src/util';
+import * as convs from './Conversions';
```

Comment:
Dont like

Reply:
what?

Reply:
convs instead of conversions. And namespacing everything...